### ISSUE TITLE
Audio Balance in accessibility does not work

### ISSUE THREAD

**Issue by rbenassirp** [PixelExperience/android-issues#728]
## Build date

```
PixelExperience_lavender-11.0-20210816-0620-OFFICIAL.zip
```

## Expected Behavior

```
It is supposed to move the audio balance dot between left and right and audio volume respect the amount moved
```

## Current Behavior

```
today you are able to move the dot but audio is always 50% 50% volume between channels
```

## Possible Solution

```
make audio balance respect the position chosen
L----------I----------R (50%L 50%R)
then you move
L----I---------------R then get (75%L and 25%R)
```

## Steps to Reproduce

```
1. turn on a music or a video
2. go to settings > acessibility > audio balance
3. move the dot
4. nothing happens
```

/device lavender (found at https://download.pixelexperience.org/lavender)
/version eleven

**Comment by pixelexperiencebot** [comment]
Validation failed, this issue will be closed for not complying with the established standards.

Errors:
- Invalid device: lavender (found at https://dow
